docs/examples/compute/omc_managed_server/README.md

```diff
@@ -0,0 +1,113 @@
+    #     ___  ____     _    ____ _     _____
+    #    / _ \|  _ \   / \  / ___| |   | ____|
+    #   | | | | |_) | / _ \| |   | |   |  _|
+    #   | |_| |  _ < / ___ | |___| |___| |___
+    #    \___/|_| \_/_/   \_\____|_____|_____|
+***
+## Bare Metal Cloud Instance managed with Oracle Management Cloud Agent
+These two configurations 
+
+It creates a VN
+
+### Using this example
+* This example is predicated on your usage of Oracle Management Cloud, and having downloaded the AgentInstall.zip and generating an Agent Registration Key
+Details can be found here - <some link>
+* Update env-vars with the required information. Most examples use the same set of environment variables so you only need to do this once.
+* Source env-vars
+  * `$ . env-vars`
+
+* Two paths are available to create an OMC managed instance. 1. The reccomended approach is a two stage deploy, where we first load the agent, and download necessary agent files and create and image that we can reuse. This approach allows you to do the time consuming work one time, and create an image to be used to configure the agent on demand.  2. Install and configure the agent in one single step.  This approach takes about 20 minutes to bring a server online.
+
+### Two  Stage Deploy
+* Create your base server with the agent installed
+* `$ cd create_omc_base_server`
+* Update examples.tfvars with your specific details
+    * Region where you want to deploy
+    * The compartment name to run your instance
+    * The OCID for the subnet where you want your instance to run.
+        * It is required that this subnet be public and allow SSH traffic
+    * The path to your AgentInstall.zip
+    * Your Agent registration key
+* Load the resource module 
+* `$ terraform get`
+* `$ terraform plan -var-file=example.tfvars`
+* `$ terraform apply -var-file=example.tfvars`
+* Once your instance is running, goto the console, and create a custom image from this server.  Note the ocid of the image that is created.
```

Comment:
if only implementing a packer provider and a terraform provider were the same task 😿